lib/screens/headways.ex

```diff
@@ -124,16 +126,27 @@ defmodule Screens.Headways do
     ],
     red_ashmont: ~w[shmnl fldcr smmnl asmnl],
     red_braintree: ~w[nqncy wlsta qnctr qamnl brntn],
-    red_trunk: ~w[alfcl davis portr harsq cntsq knncl chmnl sstat brdwy andrw jfk]
+    red_trunk: ~w[alfcl davis portr harsq cntsq knncl chmnl brdwy andrw jfk],
+    silver_chelsea: ~w[estav boxdt belsq chels]
   }
 
   # Mapping of parent station and route IDs to headway keys, for parent stations which serve more
   # than one line.
   @multi_stations %{
-    blue_trunk: {~w[Blue], ~w[state gover]},
+    blue_trunk: {~w[Blue], ~w[state gover aport]},
     green_trunk: {~w[Green-B Green-C Green-D Green-E], ~w[north haecl gover pktrm]},
     orange_trunk: {~w[Orange], ~w[north haecl state dwnxg]},
-    red_trunk: {~w[Red], ~w[pktrm dwnxg]}
+    red_trunk: {~w[Red], ~w[pktrm dwnxg sstat]},
+    silver_seaport: {~w[741 742 746], ~w[conrd wtcst crtst sstat]},
+    silver_chelsea: {~w[743], ~w[conrd wtcst crtst sstat aport]}
+  }
+
+  # Mapping of parent stops and route IDs to headway keys for the Silver Line,
+  # for parent stops which serve more than one route.
```

Comment:
We shouldn't use "parent" here as these are specifically _not_ parent stops (they are "standalone stops", as we sometimes call stopping locations with no parent).

Reply:
Follow-up: "parent" stops has been removed from this line, but not the line above ("Mapping of parent stops and...")